exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,13 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode( sentence ){
+  if ( sentence === sentence1  ){
```

Comment:
This comparison is incorrect you should use a function to check if the word 'code' is a substring of the string passed in as a parameter. You could use the 'includes' function to achieve this for exampe:

https://flaviocopes.com/how-to-string-contains-substring-javascript/